qemu/tests/cfg/virtio_fs_509_memslots.cfg

```diff
@@ -0,0 +1,74 @@
+- virtio_fs_509_memslots:
+    type = virtio_fs_509_memslots
+    no RHEL.6 RHEL.7 RHEL.8
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    no Win2008 Win7 Win2012..r2 Win8 Win8.1
+    no s390x
+    required_qemu = [9.0.0,)
+    start_vm = yes
+    hmp_timeout = 10
+
+    filesystems = fs
+    fs_driver = virtio-fs
+    fs_driver_props = {"queue-size": 1024}
+    fs_source_type = mount
+    fs_source_dir = /var/tmp/virtio_fs_test
+    force_create_fs_source = no
```

Comment:
All the fs paths are created by user, how about setting fs_source_user_config = yes, then it could save time in env_process.py

Reply:
A test file is created before test execution. So don't need create fs source again here.

Reply:
Make sense.